SpatialGDK/Source/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -134,20 +157,17 @@ FNetworkGUID FSpatialNetGUIDCache::AssignNewEntityActorNetGUID(AActor* Actor, co
 		Receiver->ResolvePendingOperations(Actor, ObjectRef);
 	}
 
-	// Allocate NetGUIDs for each subobject, sorting alphabetically to ensure stable references.
-	TArray<UObject*> ActorSubobjects;
-	GetSubobjects(Actor, ActorSubobjects);
-
-	for (UObject* Subobject : ActorSubobjects)
+	for (auto& Pair : SubobjectToOffset)
 	{
-		const uint32* Offset = SubobjectToOffset.Find(*Subobject->GetName());
-		if (Offset != nullptr)
-		{
-			FNetworkGUID SubobjectNetGUID = GetOrAssignNetGUID_SpatialGDK(Subobject);
-			FUnrealObjectRef SubobjectRef(EntityId, *Offset);
-			RegisterObjectRef(SubobjectNetGUID, SubobjectRef);
-			UE_LOG(LogSpatialOSPackageMap, Log, TEXT("Registered new object ref for subobject %s inside actor %s. NetGUID: %s, object ref: %s"),
-				*Subobject->GetName(), *Actor->GetName(), *SubobjectNetGUID.ToString(), *SubobjectRef.ToString());
+		UObject* Subobject = Pair.Key;
+		uint32 Offset = Pair.Value;
+
+		FNetworkGUID SubobjectNetGUID = GetOrAssignNetGUID_SpatialGDK(Subobject);
+		FUnrealObjectRef SubobjectRef(EntityId, Offset);
+		RegisterObjectRef(SubobjectNetGUID, SubobjectRef);
+
+		UE_LOG(LogSpatialOSPackageMap, Verbose, TEXT("Registered new object ref for subobject %s inside actor %s. NetGUID: %s, object ref: %s"),
+			*Subobject->GetName(), *Actor->GetName(), *SubobjectNetGUID.ToString(), *SubobjectRef.ToString());
 
```

Comment:
Bad tabbing below.

Reply:
Huh, it looks just fine in the actual file.